pandas/core/generic.py

```diff
@@ -7378,7 +7378,7 @@ def resample(self, rule, how=None, axis=0, fill_method=None, closed=None,
         rule : str
             The offset string or object representing target conversion.
         how : str
-            Method for down-/re-sampling, default to ‘mean’ for downsampling.
+            Method for down/re-sampling, default to ‘mean’ for downsampling.
```

Comment:
I didn't see that, but can we use regular quotes for mean. I think those are not regular single quotes, are they?

Reply:
Oh, yeah, that might be it! I couldn't see them either.